### PR TITLE
Sdks 3608 add missing docs

### PR DESCRIPTION
**Pull request by vahancouver**
[SDKS-3608](https://pingidentity.atlassian.net/browse/SDKS-3608) [iOS][Ping SDK] Missing docs for public interfaces

Also updated the podspecs and file copyrights

### CODE REVIEW

**Review by spetrov — CHANGES_REQUESTED** [review]
I found only one typo to be fixed... everything else looks very good! 👍🏻

File: Oidc/Oidc/OpenIdConfiguration.swift [hunk 1]
```diff
@@ -11,12 +11,17 @@
 
 import Foundation
 
+/// Struct epresenting the OpenID Connect configuration.
```
[the remark this answers is not in the document]
Comment (vahancouver) [reply Oidc/Oidc/OpenIdConfiguration.swift]: fixed

File: Storage/Storage/Encryptor.swift [hunk 2]
```diff
@@ -19,7 +19,7 @@ public protocol Encryptor {
   /// - Throws: An error if encryption fails.
   func encrypt(data: Data) async throws -> Data
   
-  // Decrypts the given data.
+  /// Decrypts the given data.
```
[the remark this answers is not in the document]
Comment (vahancouver) [reply Storage/Storage/Encryptor.swift:22]: @spetrov in protocol definitions we cannot specify access modifiers 
(XCODE: Protocol requirements implicitly have the same access as the protocol itself)

File: Storage/Storage/Encryptor.swift [hunk 3]
```diff
@@ -19,7 +19,7 @@ public protocol Encryptor {
   /// - Throws: An error if encryption fails.
   func encrypt(data: Data) async throws -> Data
```
[the remark this answers is not in the document]
Comment (vahancouver) [reply Storage/Storage/Encryptor.swift:20]: Same here

**Review by spetrov — APPROVED** [review]

**Pull request merged**